performance.rst

```diff
@@ -111,7 +111,7 @@ performance significantly.
 During container compilation, Symfony generates the file with the list of
 classes to preload. The only requirement is that you need to set both
 ``container.dumper.inline_factories`` and  ``container.dumper.inline_class_loader``
-parameters to ``true``.
+parameters to ``false``.
```

Comment:
Can't we remove the sentence altogether? To me, the preload file is always generated no matter the value of these parameters.

Reply:
Yes, agree that it can be removed here.
@nicolas-grekas Do you have an idea where is the best place to put a note about these flags should be `false` with preload enabled?

Reply:
`container.dumper.inline_factories` should be `true` with preloading if I understood correctly, and `container.dumper.inline_class_loader` should be `false`?

This could be added as a `::tip` note maybe?

Reply:
> `container.dumper.inline_factories` should be `true` with preloading if I understood correctly, and `container.dumper.inline_class_loader` should be `false`?

Correct

> This could be added as a ..tip note maybe?

I think it's kind of a hard requirement to have `inline_class_loader: false`, to avoid unexpected errors.

What do you think about this wording?
> The only requirement is that you need to set parameters
``container.dumper.inline_factories: true`` and  ``container.dumper.inline_class_loader: false``,
to disable performance improvements designed for execution without preload.

Let me know if it should be `Tip:` or new wording is correct.

Reply:
> I think it's kind of a hard requirement to have inline_class_loader: false, to avoid unexpected errors.

That's not a requirement, it works without this setting. If not, that should be reported and fixed.

I'm not sure anymore that we should add anything here, this is really a minor difference.
Let's just remove that sentence.

Reply:
> Let's just remove that sentence.
Ok, removed

> That's not a requirement, it works without this setting. If not, that should be reported and fixed.
Ok, I'll report it as a bug separately, as it doesn't work well with preload enabled.